src/marks/lines_number.py

```diff
@@ -1,74 +1,45 @@
 from talon import Context, Module
+from dataclasses import dataclass
 
 mod = Module()
 ctx = Context()
 
 mod.list("cursorless_line_direction", desc="Supported directions for line modifier")
 
-directions = {
-    "row": {"isRelative": False, "transformation": lambda number: number - 1},
-    "up": {"isRelative": True, "transformation": lambda number: -number},
-    "down": {"isRelative": True, "transformation": lambda number: number},
-}
 
-ctx.lists["self.cursorless_line_direction"] = directions.keys()
+@dataclass
+class CustomizableTerm:
+    defaultSpokenForm: str
+    cursorlessIdentifier: str
+    type: str
+    formatter: callable
 
 
-def parse_line(line: dict):
-    direction = directions[line["direction"]]
-    line_number = line["lineNumber"]
-    return {
-        "lineNumber": direction["transformation"](line_number),
-        "isRelative": direction["isRelative"],
-    }
-
+# NOTE: Please do not change these dicts.  Use the CSVs for customization.
+# See https://github.com/pokey/cursorless-talon/blob/main/docs/customization.md
+directions = [
+    CustomizableTerm("row", "modulo100", "modulo100", lambda number: number - 1),
+    CustomizableTerm("up", "relativeUp", "relative", lambda number: -number),
+    CustomizableTerm("down", "relativeDown", "relative", lambda number: number),
```

Comment:
Given that these are lumped into same csv with other special marks, maybe make identifier a bit more verbose?

```suggestion
    CustomizableTerm("row", "modulo100", "lineNumberModulo100", lambda number: number - 1),
    CustomizableTerm("up", "lineNumberRelativeUp", "relative", lambda number: -number),
    CustomizableTerm("down", "lineNumberRelativeDown", "relative", lambda number: number),
```

Reply:
Agreed